spec/railtie_spec.rb

```diff
@@ -27,4 +27,18 @@ def index
       expect(ActiveRecord::Relation.ancestors).to include(Readyset::RelationExtension)
     end
   end
+
+  describe 'readyset.connection_pools' do
+    it 'sets up connection pools for both the reading and writing roles' do
+      pools = ActiveRecord::Base.connection_handler.connection_pools
+      readyset_pool = pools.find { |pool| pool.shard == Readyset.config.shard }
+
+      expect(readyset_pool).not_to be_nil
```

Comment:
**_question:_** Is the `nil` expectation here just in-case the bottom two expectations turn up `nil`? :eyes:

Reply:
I just wanted to make explicit that a connection pool for Readyset should exist at that point